claat/cmd/auth.go

```diff
@@ -87,10 +87,10 @@ func driveClient() (*http.Client, error) {
 
 // tokenSource creates a new oauth2.TokenSource backed by tokenRefresher,
 // using previously stored user credentials if available.
-func tokenSource(provider string) (oauth2.TokenSource, error) {
-	// Ignore provider if *authToken is given.
-	if *authToken != "" {
-		tok := &oauth2.Token{AccessToken: *authToken}
+func tokenSource(provider, authToken string) (oauth2.TokenSource, error) {
```

Comment:
Add doc about what happens if `authToken` is the empty string please

Reply:
Where?

Reply:
In the godoc above the method preferably - mostly just for future API implementers 😄